sale_automatic_workflow/models/sale_workflow_process.py

```diff
@@ -35,36 +35,41 @@ def _default_filter(self, xmlid):
         default='direct',
     )
     validate_order = fields.Boolean(string='Validate Order')
-    order_filter_domain = fields.Char(
+    order_filter_domain = fields.Text(
         string='Order Filter Domain',
-        default="[('state', '=', 'draft')]"
+        related='order_filter_id.domain',
```

Comment:
@lasley I understand you POV. I don't know if it's already the case but I've observed that event if the content of a related field is not changed on edit, odoo call the method write on the related record. In some cases this kink of behavior can lead to security error if the user has not the right to update the target model. That's why I've the habit to declare my related fields readonly by default... But I can approve this PR as it is. :smirk: